src/restic/node.go

```diff
@@ -581,6 +581,7 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "file":
 		node.Size = uint64(stat.size())
 		node.Links = uint64(stat.nlink())
+		node.Device = uint64(stat.dev())
```

Comment:
Huh, I think you're right. Good catch! I'll prepare a PR, and add tests (so we can catch it).